flixel/FlxCamera.hx

```diff
@@ -786,6 +795,8 @@ class FlxCamera extends FlxBasic
 		updateFade(elapsed);
 		updateShake(elapsed);
 		
+		flashSprite.filters = enableFilters ? _filters : null;
```

Comment:
It should, read https://github.com/HaxeFlixel/flixel/pull/1635#discussion_r42324248 to know why it has to apply the filters each time. And if there are no filters `set_filters` only flags the `DisplayObject` to update which is already true because other modifications.